### PR TITLE
Libpitt/88 user instructions

### PR DESCRIPTION
**Pull request by libpitt**
**Change log:**
1. Add user email to header #89 
   - Refactor header code to use react bootstrap `Nav` component so as to get mobile view ui
   - Move nav related code to own component `AppNavBar`
2. Add user unauthorized message #88 
   - Add accommodating styles
   - Add `/login` link (pages/login.js)
   - Make use of `response.data` in `verifyInReadGroup` to specifically check read privs on each token check
   
### IMPORTANT:
SenNet, update to: `NEXT_PUBLIC_PRIVS_GROUP_URL = $NEXT_PUBLIC_API_BASE/privs`
HuBMAP, add: `NEXT_PUBLIC_PRIVS_GROUP_NAME = HuBMAP Read`

### CODE REVIEW

**Review by maxsibilla — APPROVED** [review]
@libpitt Message looks good for SenNet at least
image

**Comment by maxsibilla** [comment]
@yuanzhou I am tagging you as a reviewer as well on this. The main ask for this PR is 

1. Add the user's email to the navbar for clarity, consistency across the UIs, and to help us with troubleshooting account access problems
2. Add a check for the use case that a user has globus access but is not a member of the default read group (or has read access). In this case we want to show a message explaining what the user needs to do with links to the member registration page and the help desk.

**Review by yuanzhou — CHANGES_REQUESTED** [review]

File: src/components/AppLogin/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    const pageData = () => {
+        if (unauthorized) {
+            const body = <div>
+                <p>You are trying to access the {t('HuBMAP')} Data Ingest Board logged in as <strong className={robotoBold.className}>{getUserEmail()}</strong>.
+                You are not authorized to log into this application with that account.  Please check that
+                you have registered via the <a href={t('https://profile.hubmapconsortium.org/profile')} className='lnk--ic'>Member Registration Page <UserOutlined /></a>,
+                have provided Globus account information and are logging in with that account.</p>
+
+                <p>Once you have confirmed your registration information you can <a href='/login' className='lnk--ic'>log in <LoginOutlined /></a> again.</p>
```
[the remark this answers is not in the document]
Comment (libpitt) [reply src/components/AppLogin/index.js:18]: @yuanzhou It shouldn't, there is a login page. Unless misunderstanding what you're referring to...
:login-page

[the remark this answers is not in the document]
Comment (yuanzhou) [reply src/components/AppLogin/index.js:18]: I didn't check locally, on the current released version, I got 404 for both HuBMAP and SenNet
Screenshot 2023-10-17 at 11 08 29 AM

**Review by yuanzhou — APPROVED** [review]

**Pull request merged**